fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -645,6 +645,13 @@ protected boolean cancelImpl(String errMsg) {
             return false;
         }
 
+        try {
+            this.deleteTabletWatermarkTxnId =
+                    Env.getCurrentGlobalTransactionMgr().getNextTransactionId();
+        } catch (UserException e) {
+            LOG.warn("get next transaction id failed");
+        }
```

Comment:
This assign method could be better put into AlterJob.java, just add an unified method like:
```java
protected void assignDeleteTabletWatermarkTxnId() {
    try {
            this.deleteTabletWatermarkTxnId =
                    Env.getCurrentGlobalTransactionMgr().getNextTransactionId();
        } catch (UserException e) {
            LOG.warn("get next transaction id failed");
        }
}
```
and we cloud just call assignDeleteTabletWatermarkTxnId() here and also in SchemaChangeJobV2, even in CloudSchemaChangeJob and CloudRollupJob in the future.

Reply:
done